ansible/roles/jenkins-worker/tasks/partials/tap2junit/aix.yml

```diff
@@ -16,15 +16,18 @@
   when: pip_exists.stat.exists == False
 
 - name: install pip
-  raw: /usr/bin/python /home/iojs/get-pip.py
+  raw: /opt/bin/python3 /home/iojs/get-pip.py
```

Comment:
Will this work on the AIX 7.1 boxes?

```
$ ssh test-ibm-aix71-ppc64_be-4 "ls -al /opt/bin"
/opt/bin not found
$
```

Reply:
no but we shouldnt need to reinstall pip on those boxes so the step will be skipped

Reply:
hmm, we may have the AIX 6.1 boxes for a while, would be good if the ansible scripts could be used to replace one if that becomes necessary.

Reply:
ive added some logic for aix71 and aix72

Reply:
ive added some logic for aix71 and aix72